sidekiq/README.md

```diff
@@ -57,27 +57,10 @@ No additional installation is needed on your server.
        mappings:
          - match: "sidekiq.sidekiq.*"
            name: "sidekiq.*"
```

Comment:
I think this might be incorrect. What are the metrics we are trying to match ?

Reply:
Addressed in https://github.com/DataDog/integrations-core/pull/6697/files#diff-2c9bcc1f9550ef40d5cc6d48658c7e7cR58